orga/tests/acceptance/campaign-update_test.js

```diff
@@ -1,56 +1,94 @@
 import { module, test } from 'qunit';
-import { currentURL, visit } from '@ember/test-helpers';
+import { currentURL } from '@ember/test-helpers';
 import { fillByLabel, clickByName, visit as visitScreen } from '@1024pix/ember-testing-library';
 import { setupApplicationTest } from 'ember-qunit';
 import authenticateSession from '../helpers/authenticate-session';
-import { createUserWithMembershipAndTermsOfServiceAccepted, createPrescriberByUser } from '../helpers/test-init';
+import { createAdmin, createMember } from '../helpers/test-init';
 
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 
 module('Acceptance | Campaign Update', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
-  hooks.beforeEach(async () => {
-    const user = createUserWithMembershipAndTermsOfServiceAccepted();
-    createPrescriberByUser(user);
+  module('when user is ADMIN', function () {
+    test('it should allow to update a campaign and redirect to the newly updated campaign', async function (assert) {
+      // given
+      const { user } = createAdmin();
+      await authenticateSession(user.id);
+      const campaign = server.create('campaign', { id: 1 });
+      const newName = 'New Name';
+      const newText = 'New text';
 
-    await authenticateSession(user.id);
+      await visitScreen(`/campagnes/${campaign.id}/modification`);
+      await fillByLabel('* Nom de la campagne', newName);
+      await fillByLabel("Texte de la page d'accueil", newText);
+
+      // when
+      await clickByName('Modifier');
+
+      // then
+      assert.strictEqual(server.db.campaigns.find(1).name, newName);
+      assert.strictEqual(server.db.campaigns.find(1).customLandingPageText, newText);
+      assert.strictEqual(currentURL(), '/campagnes/1/parametres');
+    });
+
+    test('it should show campaign title', async function (assert) {
```

Comment:
Je me demande si ce test aurait pas du être une assertion en plus dans le test précédent qui vérifie que l'update a bien été effectué ?
`assert.dom(screen.getByText(newName)).exists();` 
Ou alors le sortir hors du module ''when user is ADMIN' ?

Reply:
ah oui, on a rajouté le test à part parce que c'était un bug qu'on a corrigé, mais on peut en effet le fusionner avec le précédent , de mon côté les deux me vont ^^

Reply:
je l'ai sorti du module ADMIN :)